API/Auth.pm

```diff
@@ -24,6 +24,10 @@ our ($serial, $cbc);
 my $cbcRetry = 30;
 my (%waiters);
 
+# Define a robust desktop User-Agent string.
+my $desktopUA = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36";
+my %headers = ( "User-Agent" => $desktopUA );
```

Comment:
Please define these headers locally in the get, as it's very specific to that call. Other than that I'd say this is looking good.